Library/Homebrew/formula_installer.rb

```diff
@@ -368,7 +368,7 @@ def check_conflicts
 
       raise if ARGV.homebrew_developer?
 
-      $stderr.puts "Please report this to the #{formula.tap} tap!"
+      $stderr.puts "Please report this issue to the #{formula.tap} tap (not Homebrew/brew or Homebrew/core)!"
```

Comment:
Maybe we could add a link to GitHub issues for a tap, what do you think?

Reply:
Unfortunately we cannot know for sure whether the tap has issues enabled.

Reply:
Fair enough, didn't think about it, thanks!